packages/core/package.json

```diff
@@ -39,7 +39,8 @@
     "react": "^16.8.2",
     "react-dom": "^16.8.2",
     "react-test-renderer": "^16.8.2",
-    "stylis": "^3.4.9"
+    "stylis": "^3.4.9",
+    "pcln-icons": "^2.0.1"
```

Comment:
Was this rearrange of `pcln-icons` dep to `devDependencies` and `peerDependencies` done manually? (Believe if you were to "_cd_" into `packages/core`, and run `npm i -D pcln-icons`, it'd have sorted the `pcln-icons` dep alphabetically.).

Can you update the `pcln-icons` as `devDependencies` systematically? For `peerDependencies`, you can still keep the alphabetic sort intact for your manual entry.

Reply:
fixed